plugins/org.jkiss.dbeaver.ui.editors.base/src/org/jkiss/dbeaver/ui/editors/EditorUtils.java

```diff
@@ -232,6 +232,8 @@ public static void setResourceProperty(@NotNull RCPProject project, @NotNull IRe
     public static DatabaseEditorContext getEditorContext(IEditorInput editorInput) {
         if (editorInput instanceof IInMemoryEditorInput) {
             return (DatabaseEditorContext) ((IInMemoryEditorInput) editorInput).getProperty(PROP_EDITOR_CONTEXT);
+        } else if (editorInput instanceof IncludedScriptFileEditorInput) {
+            return ((IncludedScriptFileEditorInput) editorInput).getDatabaseEditorContext();
```

Comment:
Use instanceof matching

```suggestion
        } else if (editorInput instanceof IncludedScriptFileEditorInput input) {
            return input.getDatabaseEditorContext();
```